src/main/msp/msp.c

```diff
@@ -4147,7 +4147,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             }
 
             const unsigned textLength = MIN(textSpace, sbufReadU8(src));
-            memset(textVar, 0, strlen(textVar));
+            memset(textVar, 0, textLength + 1);
```

Comment:
IMO it is better to clear all memory allocated for variable:
```suggestion
            memset(textVar, 0, textSpace  + 1);
```

(if you want only to 0-terminate string, this would do it:)
```suggestion
            textVar[textLength] = '\0';
```